e2e/k8s.yml

```diff
@@ -4288,18 +4288,25 @@ entities:
             - namespace
           legacyEventTypes:
             - K8sDeploymentSample
-      - name: k8s.deployment.isAvailable
+      - name: k8s.deployment.conditionAvailable
         type: string
         migrationInformation:
           legacyNames:
-            - isAvailable
+            - conditionAvailable
```

Comment:
I might be misunderstanding what legacyNames is, but would we want to keep the old attribute name under there for migration purposes?

Also, will this change break user dashboards that are dependent on the old names isAvailable and isProgressing ?

Reply:
Good questions:

> I might be misunderstanding what legacyNames is, but would we want to keep the old attribute name under there for migration purposes?
This file is used to convert, in infra backend, Samples to Dimensional Metrics (DMs) at ingest time; so usually we want to keep the old attribute name, but as explained below, this is not the case for this PR.

> Also, will this change break user dashboards that are dependent on the old names isAvailable and isProgressing ?
No, these attributes are part of the same [NRIT ticket](https://issues.newrelic.com/browse/NR-84698) that covers this PR, they have not been released so far. I changed the name to better reflect the fact that all three of them are related.

Reply:
@juanjjaramillo just to confirm what you are saying, `k8s.deployment.conditionAvailable` and `k8s.deployment.isProgressing` are metrics you had previously added, but not released? So renaming them isn't an issue?